Mage.Tests/src/test/java/org/mage/test/cards/abilities/oneshot/exile/BanishmentTest.java

```diff
@@ -0,0 +1,57 @@
+package org.mage.test.cards.abilities.oneshot.exile;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestCommander4Players;
+
+public class BanishmentTest extends CardTestCommander4Players {
```

Comment:
I actually think you should change it back to have a limited range of influence.
Then give playerD some Memnites as well.
This way you can check that the card respects range of influence since playerC's permanents won't be touched, but it will correctly exile all other opponent controlled Memnites